src/model/Model.hpp

```diff
@@ -311,6 +311,9 @@ class MODEL_API Model : public openstudio::Workspace {
       std::shared_ptr<typename T::ImplType> p = it->getImpl<typename T::ImplType>();
       if (p) { result.push_back(T(p)); }
     }
+    if (sorted) {
+      std::sort(result.begin(), result.end());
+    }
```

Comment:
add an optional argument to sort stuff in getConcreteModelObjects, defaults to false like it was before.